changelogs/unreleased/add-granularity-option.md

```diff
@@ -0,0 +1,5 @@
+## feature/core
+
+* Added granularity option to box.cfg{}. Granularity is an option
+  that allows user to set multiplicity of memory allocation in small
+  allocator. Granulatiry must be exponent of two and >= 4.
```

Comment:
Nit: as a rule we also add at the end of changelog number of issue (in brackets): (gh-5518).

Reply:
Nit: also you changed another one user-visible thing - check of alloc_factor value. I guess you should add another one entry for this change.